src/utils/adjust-markup.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const expand = require('emmet').expandAbbreviation;
```

Comment:
We could alternatively use the more lightweight [put-selector](https://github.com/kriszyp/put-selector) module, but the syntax is a bit different so might be confusing for authors that are used to emmet in their editor

Reply:
Let's keep emmet.